Tests/SentryTests/Helper/TestSysctl.swift

```diff
@@ -21,4 +21,14 @@ class TestSysctl: SentrySysctl {
     public func setProcessStartTimestamp(value: Date) {
         internalProcessStartTimestamp = value
     }
+    
+    private var internalModuleInitializationTimestamp: Date = Date(timeIntervalSinceReferenceDate: 0)
```

Comment:
m: this would always create a date of January 1 at time midnight UTC. Are we using this as a sentinel value? Why not `nil`?

Reply:
It's just a fixed value we can safely increment and predict the outcome.